pyradiosky/skymodel.py

```diff
@@ -627,10 +688,12 @@ def check(self, check_extra=True, run_check_acceptability=True):
             else:
                 allowed_units = ["Jy/sr", "K"]
 
-            if not (
-                param_unit.is_equivalent(allowed_units[0])
-                or param_unit.is_equivalent(allowed_units[1])
-            ):
+            equivalent = False
+            for unit in allowed_units:
+                if param_unit.is_equivalent(unit):
```

Comment:
I was under the impression you could pass a tuple of all units you are looking to test at once: https://docs.astropy.org/en/stable/api/astropy.units.UnitBase.html#astropy.units.UnitBase.is_equivalent 
Wouldn't need to test them all separately yourself

Reply:
Oh! I didn't know that. I'll give it a try.